modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/codegen/generator/StructGenerator.kt

```diff
@@ -1,11 +1,23 @@
 package jp.co.soramitsu.iroha2.codegen.generator
 
 import com.squareup.kotlinpoet.TypeSpec
+import jp.co.soramitsu.iroha2.GsonSerializable
 import jp.co.soramitsu.iroha2.codegen.blueprint.StructBlueprint
 
 object StructGenerator : AbstractGenerator<StructBlueprint>() {
+
+    private val manuallySerializableTypes = listOf("EvaluatesTo", "Metadata")
```

Comment:
I believe json serialisation must not affect codegeneration. For person who probably will review won't be clear why this classes has "special" treatment. So better to remove such list of "special" types